packages/utils/src/date/index.ts

```diff
@@ -1,6 +1,6 @@
 import dateFnsFormat from 'date-fns/format';
 import parse from 'date-fns/parse';
-export * from './generator';
+import * as generator from './generator';
```

Comment:
what's the difference? `export * from './generator'` will export all named exports of `generator` as I understand...

Reply:
its not resolved in the RTL tests when we consume the latest TUI components with this recent changes, 
failing here 
https://github.com/Talend/ui/blob/master/packages/components/src/DateTimePickers/generator.js#L32
because of the default export here
https://github.com/Talend/ui/blob/master/packages/utils/src/index.ts#L1